docs/agents/sematext-agent/jmx-attaching-containers.md

```diff
@@ -0,0 +1,21 @@
+title: Automatic attaching to Java processes in container environments
+description: Sematext Agent can automatically attach to Java processes (Solr, Kafka, Zookeeper...) and expose JMX port required for monitoring from within the service itself
+
+When you assign a `MONITORING_TOKEN` to a Java service (e.g. via env variable), Sematext Agent will examine the process to see whether it can be
```

Comment:
When you assign a `MONITORING_TOKEN` to a Java service (e.g. via env variable), Sematext Agent will examine the process to see whether it can be
=>
When you assign a `MONITORING_TOKEN` to a Java service, via an environment variable, Sematext Agent will examine the process to see whether it can be